src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -39,51 +45,30 @@ public void tearDown() throws Exception {
 
     @Test
     public void testFindDefaultDueDateToday() {
-        prepCuDisbursementVoucherDefaultDueDateServiceImpl("1", Calendar.getInstance());
+        prepCuDisbursementVoucherDefaultDueDateServiceImpl("1");
         Calendar expectedCalendar = Calendar.getInstance();
         expectedCalendar.add(Calendar.DAY_OF_MONTH, 1);
 
-// KFSPTS-31897 - fix failing unit test
-//        assertServiceDateEqualsExpectedDate(new Date(expectedCalendar.getTimeInMillis()));
+        assertServiceDateEqualsExpectedDate(new Date(expectedCalendar.getTimeInMillis()));
     }
     
-    @Test
-    public void testFindDefaultDueDateLeapYear() {
```

Comment:
If it's practical to implement the suggested `DateTimeService` improvements, then these removed test methods should be added back in (and updated for `java.time` compatibility).